tsdb/db.go

```diff
@@ -865,7 +865,7 @@ func openBlocks(l log.Logger, dir string, loaded []*Block, chunkPool chunkenc.Po
 	for _, bDir := range bDirs {
 		meta, _, err := readMetaFile(bDir)
 		if err != nil {
-			level.Error(l).Log("msg", "not a block dir", "dir", bDir)
+			level.Error(l).Log("msg", "not a block dir", "dir", bDir, "err", err)
```

Comment:
Hm.. In fact this can have many different reasons. We already check if it's block dir in https://github.com/prometheus/prometheus/blob/3bb715031f440eb98cd73bc0bd961becf61c855c/tsdb/db.go#L1321:6 

Can we change it to an overall error?  (: 
```suggestion
			level.Error(l).Log("msg", "failed to read meta.json for a block", "dir", bDir, "err", err)
```

Reply:
ok, thanks.